tests/test-sources/plugins/bufferlines/barbar.nix

```diff
@@ -17,7 +17,6 @@
     };
   };
 
-  # All the upstream default options of barbar
```

Comment:
This is completly unrelated (there are lot's of other changes like this in the PR)

Reply:
Sure, I removed the commit.